### PR TITLE
fix(components): [select]  input height is 2px more

### PR DESCRIPTION
**Pull request by chenxch**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by chenxch** [comment]
fix #7919 
image
disabled:
image
2px more

File: packages/components/select/src/useSelect.ts [hunk 1]
```diff
@@ -338,9 +337,7 @@ export const useSelect = (props, states: States, ctx) => {
       ) as HTMLInputElement
       const _tags = tags.value
 
-      const sizeInMap =
-        states.initialInputHeight ||
-        getComponentSize(selectSize.value || elForm.size)
+      const sizeInMap = getComponentSize(selectSize.value || elForm.size) - 2
       input.style.height =
         states.selected.length === 0
           ? `${sizeInMap}px`
```
Comment (YunYouJun) [inline packages/components/select/src/useSelect.ts:343]: I think we should write `-2` here. And add a comment that it's an inner input so reduce it by 2px.

File: packages/components/select/__tests__/select.test.ts [hunk 2]
```diff
@@ -1963,21 +1963,21 @@ describe('Select', () => {
     })
     await nextTick(nextTick)
     const wrapperEl = wrapper.find('input').element as HTMLDivElement
-    expect(wrapperEl.style.height).toEqual('40px')
+    expect(wrapperEl.style.height).toEqual('38px')
```
Comment (YunYouJun) [inline packages/components/select/__tests__/select.test.ts:1966]: I think we can test `.el-input__wrapper` directyly, instead of changing sizes.

Comment (chenxch) [reply packages/components/select/__tests__/select.test.ts:1966]: I tried it. It's `style.htight` an empty string.

Comment (YunYouJun) [reply packages/components/select/__tests__/select.test.ts:1966]: OK. I think we can write it as a loop by `for`.

Comment (chenxch) [reply packages/components/select/__tests__/select.test.ts:1966]: OK, leave it to me.

Comment (chenxch) [reply packages/components/select/__tests__/select.test.ts:1966]: You are so awesome that I took it straight to the test.

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**